From: Gordon Ball <gordon@chronitis.net>
Date: Thu, 13 Jan 2022 16:36:36 +0000
Subject: Patch out typeguard

The available version is too old for the usage here
---
 setup.cfg         | 4 ++--
 tests/__init__.py | 5 -----
 2 files changed, 2 insertions(+), 7 deletions(-)

diff --git a/setup.cfg b/setup.cfg
index 05f3dda..eec7437 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -29,10 +29,10 @@ install_requires =
 
 setup_requires = setuptools>=44; setuptools_scm[toml]>=3.4.3
 include_package_data = True
-tests_require = pytest; typeguard; pygments; littleutils
+tests_require = pytest; pygments; littleutils
 
 [options.extras_require]
-tests = pytest; typeguard; pygments; littleutils; cython
+tests = pytest; pygments; littleutils; cython
 
 [coverage:run]
 relative_files = True
diff --git a/tests/__init__.py b/tests/__init__.py
index fe28111..5f06412 100644
--- a/tests/__init__.py
+++ b/tests/__init__.py
@@ -1,9 +1,4 @@
-import os
-
 import pyximport
-from typeguard.importhook import install_import_hook
 
 pyximport.install(language_level=3)
 
-if not os.environ.get("STACK_DATA_SLOW_TESTS"):
-    install_import_hook(["stack_data"])
